### PR TITLE
Replaced httpparty gem, with http the gem to resolve acceptance test bug

### PR DESCRIPTION
**Pull request by Aoxian**

### CODE REVIEW

File: scripts/run_acceptance_tests.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
```
Comment (7hoenix) [inline scripts/run_acceptance_tests.sh]: Is this change necessary to fix the bug?

Generally I like adding `set -e` in my scripts and am just curious if the two are related or if you just noticed it wasn't here and added it.

File: http_server_spec/features/support/requests.rb [hunk 2]
```diff
@@ -1,23 +1,24 @@
-require "httparty"
+require "http"
 
 module Requests
   def self.get(path)
     base_url = "#{PROTOCOL}://#{HOSTNAME}:#{PORT}"
-    Response.new(HTTParty.get("#{base_url}#{path}", follow_redirects: false))
+    Response.new(HTTP.get("#{base_url}#{path}"))
+    # Response.new(HTTParty.get("#{base_url}#{path}", follow_redirects: false))
```
Comment (7hoenix) [inline http_server_spec/features/support/requests.rb]: Dead code.

Comment (Aoxian) [reply http_server_spec/features/support/requests.rb]: :face_palm:

... Walks away.

Thanks for catching this!

File: scripts/run_acceptance_tests.sh (hunk 1 above)
Comment (Aoxian) [reply scripts/run_acceptance_tests.sh]: Intentional work around because acceptance tests would randomly fail.  Now that this bug is fixed, the acceptance suite is quality and “set -e” should be here.
